src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -201,6 +201,137 @@ When the Scheduler is restarted, all the AWS instances that were previously crea
 
 
 
+===== OpenStack Infrastructure
+
+To use the OpenStack instances as computing nodes, a specific OpenStack Infrastructure
+can be created using the Resource Manager. This section describes briefly how to make it.
+
+1.  First, you need to have an admin account on your OpenStack server
+
+2.  Use the login and password information to fill in the properties *Openstack_Username*,
+    *Openstack_Username* in the *Create Node Source* panel located in the resource manager.
```

Comment:
*Openstack_Username* is repeated twice. Besides, properties are named using snake case and capitals whereas with EC2 Infrastructure it is using camel case.